composer.json

```diff
@@ -13,11 +13,22 @@
         "symfony/finder": "~2.4",
         "symfony/yaml": "~2.4"
     },
+    "require-dev": {
+        "phpunit/phpunit": "~4.8"
+    },
     "autoload": {
         "psr-0": {
             "Openl10n\\Cli": "src"
         }
     },
+    "scripts": {
+        "test": [
+            "bin/phpunit"
```

Comment:
No need to prefix by `bin/`. Composer automatically add the `bin-dir` into the `PATH` when executing commands.